README.md

```diff
@@ -104,81 +105,68 @@ navigator.login.setStatus("logged-in", {
 
-```js
-try {
-  const {profile: {id, name, email, picture}} = await navigator.credentials.get({
-  	identity: {providers: [{url: "https://example.com"}]}
+```
+const credential = await navigator.credentials.get({
+  identity: {providers: [{configURL: "https://example.com/fedcm.json"}]}
   });
-  //  ... Post a message to the IdP iframe with the account ID
-catch (e) {
-  // User either manually declined the prompt or the user was logged-out;
-  // importantly, these two outcomes are indistinguishable to the RP.
-}
-
-// Browser checks the login status for https://example.com; if there's a logged-in status with account information (that isn't expired), prompt the user with that account information. If not, issue a non-credentialled request to "https://example.com/.well-known/web-identity"; since this doesn't return the correct status+MIME type in this case, the browser is able to degrade gracefully: in passive mode, without a logged in account, the prompt is simply dismissed and an exception is thrown.
-...
-
-// In the IdP iframe, after receiving a postMessage from the toplevel document with the account ID:
-await navigator.requestStorageAccess(); // Since the user selected an account already, this will be autogranted.
-const token = (await fetch("https://example.com/login?rp=rp.example&id=" + id)).text();
-// Do whatever you would usually do with your token here.
 ```
 
-This approach works well with the Passive mode: if the user is not already signed in to the IdP, the browser doesn’t need to display anything. In Active Mode, the RP may wish to provide some guidance or trigger some action to get the user to refresh their IdP session, like navigating to the login page if it is known.
+## FedCM Optional Endpoints
 
-> [!NOTE]
-> We refer to a `url` field on the RP-side provider configurations instead of `configURL`; this is to align with future work deprecating the 
-> `configURL` parameter in favor of `url` in FedCM.
+Some IdPs may not require all aspects of the FedCM flow in order to provide a useful experience to users and RPs. It might be useful for an IdP to not define some endpoints in the [IdentityProviderAPIConfig](https://w3c-fedid.github.io/FedCM/#dictdef-identityproviderapiconfig), rather than implementing stubs. If they define none of them, no IdP configuration should be required at all.
```

Comment:
```suggestion
Some IdPs may not require all aspects of the FedCM flow to provide a useful experience to users and RPs. It might be useful for an IdP to not define some endpoints in the [IdentityProviderAPIConfig](https://w3c-fedid.github.io/FedCM/#dictdef-identityproviderapiconfig), rather than implementing stubs. If they define none of these endpoints, no IdP configuration should be required at all.
```